clients/cobol-lsp-vscode-extension/src/api/model/AnalysisResult.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import { Location } from "vscode";
+
+export class AnalysisResult {
+    public paragraphRange: Map<string, Location[]>;
+    public paragraphUsages: Map<string, Location[]>;
+    public sectionRange: Map<string, Location[]>;
+    public sectionUsages: Map<string, Location[]>;
```

Comment:
The result is something in-between state. It's not a full AnalysisResult received from the Java side and not a structure that CCF expects from our plugin. I'll reach you personally to discuss.